tests/measurements/test_var.py

```diff
@@ -35,15 +62,18 @@ def circuit(x):
 
         x = 0.54
         res = circuit(x)
-        expected = np.sin(x) ** 2
+        expected = [np.sin(x) ** 2, np.sin(x) ** 2] if isinstance(shots, list) else np.sin(x) ** 2
 
-        assert np.allclose(res, expected, atol=tol, rtol=0)
+        assert np.allclose(res, expected, atol=0.05, rtol=0.05)
         assert res.dtype == r_dtype
 
-    def test_not_an_observable(self):
+        custom_measurement_process(dev, spy)
```

Comment:
Ok so is the point of adding the call to `custom_measurement_process` that we want to make sure the old and the new functions behave the same? Because I would think as long as the original assert passes we should be replicating the behaviour and verifying the "correct" answer.

Reply:
As mentioned above, currently when executing a circuit with `qml.var`, the `QubitDevice.var` method is used. Calling `custom_measurement_process` we make sure that `QubitDevice.var` and `_Variance.process_samples`/`process_state` return the same values.